tools/eval_metric.py

```diff
@@ -146,9 +152,9 @@ def eval_metric(args):
 
 
 @torch.no_grad()
-def evaluate(model, data_loader, coco_evaluator, device, print_freq):
+def evaluate(model, data_loader, coco_evaluator, device, print_freq, args):
     # COCO evaluation
-    metric_logger = MetricLogger(delimiter="  ")
+    metric_logger = MetricLogger(args, delimiter="  ")
```

Comment:
Hi @SauravMaheshkar , Could we pass in the specific parameters to be used instead of the args here?